### PR TITLE
Feature/wil/job data

### PR DESCRIPTION
**Pull request by wil-vanderstaay**
# 
## Proposed changes

A python file to allow frontend to retrieve the UI issue results as a json object for a job.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] DevOps related

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [project guidelines](https://github.com/Visual-Testing-for-Android-Apps/Project-guidelines) doc
- [x] I locally linted my code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by alexdelicato** [comment]
Is this meant to have failed the SAM validation checks? Also there seems to be an issue with the .yml file (don't think line 62 is meant to be there?) 
image

File: server/jobData/jobData.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    table = boto3.resource("dynamodb").Table(TABLE_NAME)
+    data = table.get_item(Key={"id": LOOKUP_BATCH_ID})
+
+    # check if job id exists
+    try:
+        data["Item"]
+    except KeyError:
+        return {
+            "statusCode": 404,  # job id not found
+            "body": "{}",
```
Comment (alexdelicato) [inline server/jobData/jobData.py]: might be helpful to add the error message to the returned body

**Review by alexdelicato — APPROVED** [review]

**Comment by lgraa** [comment]
> Is this meant to have failed the SAM validation checks? Also there seems to be an issue with the .yml file (don't think line 62 is meant to be there?) image

I think it's supposed to be there. arn:aws:s3:::${SrcBucket}/* looks to have been added by Update/peter/integration, and it also appears on other lines like 111.
Looking at the log, it seems to be caused by the "npm run fast-compile" part of reportGenerator's makefile failing. I'm not sure why, since it works on my computer.

File: server/jobData/jobData.py (hunk 1 above)
Comment (reteppeter) [reply server/jobData/jobData.py]: I also think this should be changed to return an informative error message.

**Review by reteppeter — APPROVED** [review]

**Review by reteppeter — APPROVED** [review]

**Pull request merged**